src/ci/scripts/install-msys2-packages.sh

```diff
@@ -13,6 +13,4 @@ if isWindows; then
     # one way or another. The msys interpreters seem to have weird path conversions
     # baked in which break LLVM's build system one way or another, so let's use the
     # native version which keeps everything as native as possible.
-    cp C:/Python27amd64/python.exe C:/Python27amd64/python2.7.exe
-    ciCommandAddPath "C:\\Python27amd64"
```

Comment:
If we're deleting this we should also delete the comment that talks about it?

Reply:
Absolutely! Doing it right away.